### ISSUE TITLE
Forms.Legend: Does not display without height/widthrequest

### ISSUE THREAD

**Issue by jNery** [Esri/arcgis-toolkit-dotnet#233]
```XML
<Grid>
    <esri:MapView x:Name="MyMapView" Map="https://www.arcgis.com/home/webmap/viewer.html?webmap=f1ed0d220d6447a586203675ed5ac213"/>
    <toolkit:Legend x:Name="MyLegend" 
                    BindingContext="{x:Reference MyMapView}"
                    GeoView="{Binding}"
                    BackgroundColor="Black"
                    VerticalOptions="StartAndExpand"/> 
</Grid>
```

**Comment by osamaelhosany** [comment]
Hi, @jNery 
I know it's a very long time,
I Added legend control From C# not Xaml and worked fine with me 

```
private void CreateLegend() 
{
            Legend templegend = new Legend();
            templegend.BindingContext = this.esriMap; // esrimap is my mapView
            templegend.SetBinding(Legend.GeoViewProperty, ".", BindingMode.TwoWay);
            templegend.HorizontalOptions = LayoutOptions.FillAndExpand;
            templegend.VerticalOptions = LayoutOptions.FillAndExpand;
            templegend.ReverseLayerOrder = true;
            templegend.FilterByVisibleScaleRange = true;
            templegend.BackgroundColor = Color.Transparent;
            legendgrid.Children.Add(templegend); // legendgird is my parent grid 
}
```

**Comment by dotMorten** [comment]
Fixed in #291
